tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraCalendarEventAcceptMethodContract.scala

```diff
@@ -1191,15 +1180,17 @@ trait LinagoraCalendarEventAcceptMethodContract {
   }
 
   @Test
-  def shouldSucceedWhenCalendarBlobsComeFromDifferentMessages(server: GuiceJamesServer): Unit = {
-    setupServer(server, DEFAULT_INVITATION_DATA)
+  def shouldSucceedWhenCalendarBlobsComeFromDifferentMessages(server: GuiceJamesServer, eventInvitation: EventInvitation): Unit = {
```

Comment:
This test is flaky. The reason is that sometimes `blobIdFromMessage1` and `blobIdFromMessage2` are equal. The root cause is `searchReceiverInboxForNewMessages` in `LinagoraCalendarEventMethodContractUtilities` that is supposed to return the latest message that was added to user inbox but it does not always do that. Please help!

Reply:
Need to check